template/snippet.js

```diff
@@ -68,7 +68,7 @@
     var script = document.createElement('script');
     script.type = 'text/javascript';
     script.async = true;
-    script.src = 'https://<%= settings.host %><%= settings.ajsPath %>';
+    script.src = "https://<%= settings.host %><%= settings.ajsPath %>";
```

Comment:
Using double quote so that `snippet.max` and `snippet.min` would produce the same quotes. Previously, `snippet.min` would generate code that looks like `"/analytics.js/v1' + key + '/analytics.min.js"` which makes `key` a string instead of a variable.